code/modules/surgery/ear_surgery.dm

```diff
@@ -1,7 +1,6 @@
 //Head surgery to fix the ears organ
 /datum/surgery/ear_surgery
 	name = "Ear surgery"
-	requires_bodypart_type = NONE
```

Comment:
would prefer a `monkestation/code/modules/surgery/ear_surgery.dm` with something like this:

```
/datum/surgery/ear_surgery
	requires_bodypart_type = BODYTYPE_ORGANIC
```